src/node_api.cc

```diff
@@ -3088,6 +3088,18 @@ napi_status napi_get_version(napi_env env, uint32_t* result) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_node_version(napi_env env,
+                                  uint32_t* versions,
+                                  const char** release) {
```

Comment:
Thanks @addaleax, you're right about the `const char *` being a good indicator, I hadn't really considered that the object would not be a valid argument to `free` with the `const` modifier.  It does seem like all usages of this function would at least yield a global string value, so I think my original concern is invalid.  I'll take a look at the documentation and suggest any specific changes there.